shared/src/messages.cpp

```diff
@@ -10,116 +10,29 @@
 #include <stdexcept>
 
 constexpr std::string MSG_HEADER = "AIRv1.0";
-constexpr std::string START_MSG = "SM";
-constexpr std::string END_MSG = "EM";
+constexpr std::string UNSUPPORTED = "UN";
 
-bool check_line1(const std::string &line1, std::string &r_id);
-bool check_line3(const std::string &line3, std::string &c_id);
-
-std::string format_message(const msg_t &msg) {
-	std::string str_msg(MSG_HEADER + " " + msg.receiver_id + " " + START_MSG +
-						"\n" + msg.body + "\n" + END_MSG + " " + msg.caller_id);
-	return str_msg;
+bool validate_header(const std::string &str) {
+	if (str == MSG_HEADER)
+		return true;
+	return false;
 }
 
-bool check_line1(const std::string &line1, std::string &r_id) {
-	std::istringstream parts(line1);
-	std::string token;
-
-	// Protocol header
-	parts >> token;
-	if (parts.eof() || token != MSG_HEADER) {
+bool validateId(const std::string &str) {
```

Comment:
Three cases:
- ID is good
- ID is bad
- Unsupported

Don't use bool here because 3 states